website/docs/ref/core.md

```diff
@@ -115,16 +115,32 @@ Formatting of messages as strings (e.g: `"My name is {name}"`) works in developm
 The same example would in real application look like this:
 
 ```ts
-import { i18n } from "@lingui/core"
+import { i18n } from "@lingui/core";
 
 // File generated by `lingui compile`
-import { messages: messagesEn } from "./locale/en/messages.js"
+import { messages as messagesEn } from "./locale/en/messages.js";
 
-i18n.load('en', messagesEn)
+i18n.load("en", messagesEn);
 ```
 
 :::
 
+### `i18n.setMessagesCompiler(compiler)` {#i18n.setMessagesCompiler}
+
+Registers a `MessageCompiler` to enable the use of uncompiled catalogs at runtime.
+
+In production builds, the `MessageCompiler` is typically excluded to reduce bundle size.
+
+By default, message catalogs should be precompiled during the build process. However, if you need to compile catalogs at runtime, you can use this method to set a message compiler.
+
+Example usage:
+
+```ts
+import { compileMessage } from "@lingui/message-utils/compileMessage";
```

Comment:
It's coming from `@lingui/` scope, isn't it obvious enough? More over, this method wasn't built to make it possible to override a default implementation, so it's intended to use always with a default implementation.

Reply:
> this method wasn't built to make it possible to override a default implementation, so it's intended to use always with a default implementation.

That's a bit confusing to me because then I don't understand the use of this so much. But it's probably because other changes need to be done before this can be properly documented, as you said before.

> isn't it obvious enough?

Well, just a few moments ago it felt obvious to me that I can use also something else other than the default implementation, so the word "obvious" can be a bit tricky.

If no other implementation should be used, the docs should say that imho.

Anyways, I don't want to block this PR from merging. 🙂👍